safekeeper/src/wal_backup.rs

```diff
@@ -46,6 +45,14 @@ pub struct WalBackupTaskHandle {
     handle: JoinHandle<()>,
 }
 
+impl WalBackupTaskHandle {
+    pub(crate) async fn join(self) {
+        if let Err(e) = self.handle.await {
+            warn!("WAL backup task panicked: {}", e);
```

Comment:
nit: `error`

Reply:
This was an existing log line that got transplanted, but agree it should be error -- changed in 04a53a44ddaaad75656570b913d1d11d7e43ad8d